docusaurus-search-local/tsconfig.json

```diff
@@ -5,7 +5,10 @@
     "esModuleInterop": true,
     "importHelpers": true,
     "jsx": "preserve",
-    "moduleResolution": "node",
+    // moduleResolution and module are required because Docusaurus uses 'exports' in its package.json
+    // See here for more information: https://github.com/microsoft/TypeScript/issues/33079#issuecomment-1169377037
+    "moduleResolution": "Node16",
+    "module": "Node16",
```

Comment:
👍 Thanks, I tried upgrade docusaurus to rc.1, but failed, I believe this is the magic here :)